plugins/package-managers/gradle-plugin/src/main/kotlin/OrtModelBuilder.kt

```diff
@@ -52,9 +51,6 @@ internal class OrtModelBuilder : ToolingModelBuilder {
 
     private val platformCategories = setOf("platform", "enforced-platform")
 
-    private val visitedDependencies = mutableSetOf<ModuleComponentIdentifier>()
-    private val visitedProjects = mutableSetOf<ModuleVersionIdentifier>()
```

Comment:
Do you know why the previous logic kept track of visited projects / dependencies separately and handled these separately?

Reply:
IIRC simply because these two sets tracked objects of different types. Back then, for whatever reason, it did not occur to me that these have a common base type.

Reply:
Oliver is on vacation currently. Anyway, the new logic [matches what the legacy Gradle analyzer does](https://github.com/oss-review-toolkit/ort/blob/3a1fbf61a951eaed3a6450a125f278967d2155e3/plugins/package-managers/gradle/src/main/resources/init.gradle#L323), and is unrelated to the dependency graph being used. Also, the test results match those of legacy Gradle analyzer fully now (module cases where the Gradle Inspector is able to get more metadata).